### PR TITLE
Ce 921 mobile UI flow

### PR DESCRIPTION
**Pull request by Schrammel**
## Description

all of our react flow diagrams can't fit content on the right way (it is cutting labels of shapes),  and for this reason we was setting a padding, but it create a unnecessary whitespace around the flow 

Screen Shot 2022-11-07 at 13 42 42

In this PR i fixed the problem of shapes and reorganized disposition inside every flow, and now we can apply fit content efficiently
Screen Shot 2022-11-07 at 13 37 50

### CODE REVIEW

**Review by psgb-r3763 — DISMISSED** [review]
looking good, some aesthetic changes (already sent) are required to keep it looking more consistent

**Review by psgb-r3763 — CHANGES_REQUESTED** [review]
looks good! few issues though

1. desktop to mobile component transition does not happen at the correct width (should match develop)
2. desktop menu buttons should be centered in the container (they are left aligned in the pr, should match develop)
3. there could be more padding around the inner contents in desktop view, it creates a separation between the container and contents (a little less than develop)

basically the larger component should behave as it does in alpha, and look mostly the same

you can look at the develop branch for the code

**Review by psgb-r3763 — APPROVED** [review]
